src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/ChainPal.cs

```diff
@@ -49,6 +49,11 @@ public static bool ReleaseSafeX509ChainHandle(IntPtr handle)
                 verificationTime = verificationTime.ToLocalTime();
             }
 
+            // Until we support the Disallowed store, ensure it's empty (which is done by the ctor)
+            using (new X509Store(StoreName.Disallowed, StoreLocation.CurrentUser, OpenFlags.ReadOnly))
+            {
+            }
```

Comment:
Nit: `using` is fine, but you could also just do `new X509Store(...).Dispose();`.